parquet-converter failing to build

The spec that I am trying to build is:

```
spack spec -Il parquet-converters
Input spec
--------------------------------
     parquet-converters

Concretized
--------------------------------
     uwwhxrq  parquet-converters@0.1%gcc@6.4.0 build_type=RelWithDebInfo arch=linux-rhel7-x86_64
[+]  pqgtkis      ^arrow@0.11.0%gcc@6.4.0 build_type=Release +parquet~python arch=linux-rhel7-x86_64
[+]  axz6nzw          ^boost@1.68.0%gcc@6.4.0+atomic+chrono~clanglibcpp cxxstd=default +date_time~debug+exception+filesystem+graph~icu+iostreams+locale+log+math~mpi+multithreaded~numpy patches=2ab6c72d03dec6a4ae20220a9dfd5c8c572c5294252155b85c6874d97c323199 +program_options~python+random+regex+serialization+shared+signals~singlethreaded+system~taggedlayout+test+thread+timer~versionedlayout+wave arch=linux-rhel7-x86_64
[+]  7k2pzlu              ^bzip2@1.0.6%gcc@6.4.0+shared arch=linux-rhel7-x86_64
[+]  w43e56t              ^zlib@1.2.11%gcc@6.4.0+optimize+pic+shared arch=linux-rhel7-x86_64
[+]  fp22ezl          ^cmake@3.12.2%gcc@6.4.0~doc+ncurses+openssl+ownlibs patches=dd3a40d4d92f6b2158b87d6fb354c277947c776424aa03f6dc8096cf3135f5d0 ~qt arch=linux-rhel7-x86_64
[+]  cemw2y3              ^ncurses@5.9%gcc@6.4.0 patches=f84b2708a42777aadcc7f502a261afe10ca5646a51c1ef8b5e60d2070d926b57 ~symlinks~termlib arch=linux-rhel7-x86_64
[+]  6hoyzms              ^openssl@1.0.2k%gcc@6.4.0+systemcerts arch=linux-rhel7-x86_64
[+]  2nsezdo          ^flatbuffers@1.8.0%gcc@6.4.0 build_type=Release arch=linux-rhel7-x86_64
[+]  ru4pi5l          ^rapidjson@1.1.0%gcc@6.4.0 build_type=RelWithDebInfo arch=linux-rhel7-x86_64
[+]  bcgxgbw          ^snappy@1.1.7%gcc@6.4.0 build_type=RelWithDebInfo patches=c9cfecb1f7a623418590cf4e00ae7d308d1c3faeb15046c2e5090e38221da7cd +pic~shared arch=linux-rhel7-x86_64
[+]  hbbgnva          ^zstd@1.3.0%gcc@6.4.0+pic arch=linux-rhel7-x86_64
[+]  fbol32h      ^hdf5@1.10.3%gcc@6.4.0~cxx~debug~fortran~hl+mpi+pic+shared~szip~threadsafe arch=linux-rhel7-x86_64
[+]  eaj7o4x          ^hpe-mpi@2.16%gcc@6.4.0 arch=linux-rhel7-x86_64
[+]  3cqrnih      ^highfive@1.6%gcc@6.4.0~boost build_type=RelWithDebInfo +mpi arch=linux-rhel7-x86_64
[+]  oisanm6      ^synapsetool@0.3.2%gcc@6.4.0 build_type=RelWithDebInfo +mpi+shared arch=linux-rhel7-x86_64
[+]  j5wcsvb          ^python@2.7.15%gcc@6.4.0+dbm~optimizations patches=123082ab3483ded78e86d7c809e98a804b3465b4683c96bd79a2fd799f572244 +pic+pythoncmd+shared~tk~ucs4 arch=linux-rhel7-x86_64
[+]  3xpmush              ^gdbm@1.14.1%gcc@6.4.0 arch=linux-rhel7-x86_64
[+]  hrbico4                  ^readline@7.0%gcc@6.4.0 arch=linux-rhel7-x86_64
[+]  6yju4qc              ^pkg-config@0.27.1%gcc@6.4.0+internal_glib patches=49ffcd644e190dc5efcb2fab491177811ea746c1a526f75d77118c2706574358 arch=linux-rhel7-x86_64
[+]  mi3kep4              ^sqlite@3.23.1%gcc@6.4.0~functions arch=linux-rhel7-x86_64
```

And error I am getting is:

```
==> Error: ProcessError: Command exited with status 1:
    'cmake' '/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/HOME_DIR/spack/var/spack/stage/parquet-converters-0.1-uwwhxrqzyeqojemosh74udtflhgmifg6/ParquetConverters' '-G' 'Unix Makefiles' '-DCMAKE_INSTALL_PREFIX:PATH=/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/parquet-converters-0.1-uwwhxr' '-DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo' '-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON' '-DCMAKE_INSTALL_RPATH_USE_LINK_PATH:BOOL=FALSE' '-DCMAKE_INSTALL_RPATH:STRING=/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/parquet-converters-0.1-uwwhxr/lib;/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/parquet-converters-0.1-uwwhxr/lib64;/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/boost-1.68.0-axz6nz/lib;/usr/lib;/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/zlib-1.2.11-w43e56/lib;/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/rapidjson-1.1.0-ru4pi5/lib;/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/snappy-1.1.7-bcgxgb/lib;/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/zstd-1.3.0-hbbgnv/lib;/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/hdf5-1.10.3-fbol32/lib;/opt/hpe/hpc/mpt/mpt-2.16/lib;/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/python-2.7.15-j5wcsv/lib;/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/gdbm-1.14.1-3xpmus/lib;/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/readline-7.0-hrbico/lib;/usr/lib;/usr/lib;/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/sqlite-3.23.1-mi3kep/lib;/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/arrow-0.11.0-pqgtki/lib64;/usr/lib64;/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/flatbuffers-1.8.0-2nsezd/lib64;/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/synapsetool-0.3.2-oisanm/lib64;/usr/lib64;/usr/lib64' '-DCMAKE_PREFIX_PATH:STRING=/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/highfive-1.6-3cqrni;/opt/hpe/hpc/mpt/mpt-2.16;/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/hdf5-1.10.3-fbol32;/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/snappy-1.1.7-bcgxgb;/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/arrow-0.11.0-pqgtki;/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/synapsetool-0.3.2-oisanm;/gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/cmake-3.12.2-fp22ez' '-DCMAKE_C_COMPILER=/opt/hpe/hpc/mpt/mpt-2.16/bin/mpicc' '-DCMAKE_CXX_COMPILER=/opt/hpe/hpc/mpt/mpt-2.16/bin/mpicxx' '-DNEURONPARQUET_USE_MPI=ON'

1 error found in build log:
     24    --   system
     25    -- Found MPI_C: /opt/hpe/hpc/mpt/mpt-2.16/bin/mpicc (found version "3.1")
     26    -- Found MPI_CXX: /opt/hpe/hpc/mpt/mpt-2.16/bin/mpicxx (found version "3.1")
     27    -- Found MPI: TRUE (found version "3.1")
     28    -- HDF5: Using hdf5 compiler wrapper to determine C configuration
     29    -- Found HDF5: /gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-deploy/workspace/deployment/install
           /linux-rhel7-x86_64/gcc-6.4.0/hdf5-1.10.3-fbol32/lib/libhdf5.so;/gpfs/bbp.cscs.ch/apps/hpc/test/
           kumbhar/spack-deploy/workspace/deployment/install/linux-rhel7-x86_64/gcc-6.4.0/zlib-1.2.11-w43e5
           6/lib/libz.so;/usr/lib64/libdl.so;/usr/lib64/libm.so (found version "1.10.3")
  >> 30    CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
     31    Please set them or make sure they are set and tested correctly in the CMake files:
     32    PARQUET_LIB
     33        linked by target "TouchParquet" in directory /gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-d
           eploy/workspace/HOME_DIR/spack/var/spack/stage/parquet-converters-0.1-uwwhxrqzyeqojemosh74udtflh
           gmifg6/ParquetConverters/lib
     34        linked by target "CircuitParquet" in directory /gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack
           -deploy/workspace/HOME_DIR/spack/var/spack/stage/parquet-converters-0.1-uwwhxrqzyeqojemosh74udtf
           lhgmifg6/ParquetConverters/lib
     35    THRIFT_LIB
     36        linked by target "TouchParquet" in directory /gpfs/bbp.cscs.ch/apps/hpc/test/kumbhar/spack-d
           eploy/workspace/HOME_DIR/spack/var/spack/stage/parquet-converters-0.1-uwwhxrqzyeqojemosh74udtflh
           gmifg6/ParquetConverters/lib

See build log for details:
```

Is this introduced after #113  ?

--- a/var/spack/repos/builtin/packages/parquet-converters/package.py
+++ b/var/spack/repos/builtin/packages/parquet-converters/package.py
@@ -34,7 +34,7 @@ class ParquetConverters(CMakePackage):
     url      = "ssh://bbpcode.epfl.ch/building/ParquetConverters"
 
     version('develop', git=url)
-    version('0.1', git=url, tag='v0.1', preferred=True)
+    version('0.2', git=url, tag='v0.2', preferred=True)
 
     depends_on('hdf5+mpi')
     depends_on('highfive+mpi')

